src/components/Layout/Layout.js

```diff
@@ -29,6 +29,22 @@ const Wrapper = styled.div`
   }
 `;
 
+const MainContentWrapper = styled.div`
+  width: 100%;
+  flex-grow: 1 1 auto;
```

Comment:
This is invalid attribute.
flex-glow:  1;
or
flex: 1 1 auto;

and this applied to flex item not a flex container.